extmod/vfs_fat.c

```diff
@@ -38,6 +38,7 @@
 #include "timeutils.h"
 
 #define mp_obj_fat_vfs_t fs_user_mount_t
+mp_obj_t fatfs_umount(mp_obj_t bdev_or_path_in);
```

Comment:
This prototype should be moved to fsusermount.h (otherwise you get a build error when building unix)